internal/operator/version.go

```diff
@@ -2,39 +2,17 @@ package operator
 
 import (
 	"context"
-	configv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/tools/clientcmd"
-	"os"
 )
 
-// getConfigClient returns a OpenShift Config clientset used to get the ClusterVersion resource
-// in order to determine the version of the OpenShift cluster used during opcap run
-func getConfigClient() *configv1.ConfigV1Client {
-	// create openshift config clientset
-	cfg, err := clientcmd.BuildConfigFromFlags("", os.Getenv("KUBECONFIG"))
-	if err != nil {
-		logger.Errorf("Unable to build config from flags: %s", err)
-	}
-
-	clientset, err := configv1.NewForConfig(cfg)
-	if err != nil {
-		logger.Errorf("Unable to create new OpenShift Config clientset: %s", err)
-	}
-
-	return clientset
-}
-
 // GetOpenShiftVersion uses the OpenShift Config clientset to get a ClusterVersion resource which has the
 // version of an OpenShift cluster
 func (c operatorClient) GetOpenShiftVersion() (string, error) {
 	// version is the OpenShift version of the cluster
 	var version string
 
-	// OpenShift Config client used to talk with the OpenShift API
-	configClient := getConfigClient()
-
-	cversions, err := configClient.ClusterVersions().List(context.TODO(), metav1.ListOptions{})
+	cversions, err := c.KubeClientSet.ClusterVersions().List(context.TODO(), metav1.ListOptions{})
```

Comment:
@itroyano have you tested this change? I do not believe this will work due to ClusterVersions not being a native K8s type/object/resource.

Reply:
Disregard I see where this is connected and I left comment there.

Reply:
It actually worked fetching the OpenShift version.
I'll need to make a new SNO to get the exact output though, since my cluster's gone.

however I'm not entirely sure why we are doing this to begin with...  doesn't look like we're using the fetched version anywhere after it comes back to root.go...
and yeah this will produce an error when run vs. a native k8s cluster.
@acmenezes ?

Reply:
Hi @itroyano , how are we with this? Do you need any help? Let's take one step at a time. Identify where the first error was coming and fix that one. That will do a PR. Then we can discuss all the reasons behind each one. Some clients will be used multiple times and may be called behind the scenes in multiple tests others may have a single time use. Some make sense aggregating on the opcap interface others don't. They don't need to be available all the time. The use of the opcap Client interface and its field Client as well as this whole design idea is something that deserves a wider discussion since it may change completely.